setup.py

```diff
@@ -1,3 +1,12 @@
 from setuptools import setup
 
-setup()
+from peepingtom import __version__
+
+setup(
+    name='peepingtom',
+    version=f'{__version__}',
+    entry_points='''
+        [console_scripts]
+        peep=peepingtom.entry_points.peep:cli
+    '''
+)
```

Comment:
We should do this in setup.cfg like the rest, it's less finnicky!

Reply:
[example here](https://github.com/napari/napari/blob/1d784cf8373495d9591594b6dd6ac479d5566ed1/setup.cfg#L87-L89)